app/code/Magento/MediaGallery/Test/Unit/Model/DataExtractorTest.php

```diff
@@ -122,7 +144,7 @@ public function assetProvider()
             ],
             'Keyword conversion without interface' => [
                 Keyword::class,
-                null,
+                '',
```

Comment:
this value should be either a valid interface name or null

Reply:
As per function argument type declarations for the method is string so instead of null I just replaced with the empty string

Reply:
Let's keep it logically correct: it should be either interface name as string or null